src/lib.rs

```diff
@@ -1,8 +1,10 @@
 mod api_client;
+mod cli;
 mod error;
 mod serde;
 mod types;
 
 pub use api_client::*;
+pub use cli::*;
```

Comment:
this can be fine for development but before merging I'd only want to export things that *must* be in the public interface